wordle-web/app.vue

```diff
@@ -6,14 +6,52 @@
           <v-icon color="secondary"> mdi-book </v-icon>
```

Comment:
I would move the background image class to this file and add the background class right in the <v-app> so it can be the background of all pages, might need to change the color of some of the words on the pages so they stand out but could give you a central theme